### PR TITLE
Initial version using latest spring-boot-template

### PR DESCRIPTION
**Pull request by JamesCollettCGI**
### JIRA link ###
https://tools.hmcts.net/jira/browse/CCD-3662

### Change description ###
Initial version using latest spring-boot-template.

**Does this PR introduce a breaking change?**
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: Jenkinsfile_nightly [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+properties([
+  // H allow predefined but random minute see https://en.wikipedia.org/wiki/Cron#Non-standard_characters
+  pipelineTriggers([cron('H 05 * * *')])
+])
+
+@Library("Infrastructure")
+
+def type = "java"
+def product = "rpe"
+def component = "spring-boot-template"
```
[the remark this answers is not in the document]
Comment (kiran-yenigala-hmcts) [reply Jenkinsfile_nightly]: done

File: charts/ccd-migration/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+apiVersion: v2
+appVersion: "1.0"
+description: A Helm chart for ccd-migration App
+name: ccd-ccd-migration
```
[the remark this answers is not in the document]
Comment (kiran-yenigala-hmcts) [reply charts/ccd-migration/Chart.yaml]: done

File: lib/applicationinsights.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
+  "role": {
+    "name": "rpe-demo"
```
[the remark this answers is not in the document]
Comment (kiran-yenigala-hmcts) [reply lib/applicationinsights.json]: done

**Review by kiran-yenigala-hmcts — DISMISSED** [review]

**Review by Thor-tech-of-metal — APPROVED** [review]

**Review by nathan-clark — COMMENTED** [review]
Do we need the multi-arch docker image on this? and do we need travis?

**Pull request merged**